### PR TITLE
Dm/sc 4992

### PR DESCRIPTION
**Pull request by d3n153**
# Description
Extended the existing testcase @adminLogin with three more steps.

## Links to Tickets or other pull requests
https://ticketsystem.schul-cloud.org/browse/SC-4992
https://ticketsystem.schul-cloud.org/browse/SC-4824

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -9,9 +9,9 @@
             "request": "launch",
             "name": "Launch Program",
             "program": "${workspaceFolder}/index.js",
-            "args": ["-d","-t","@newsIsNotVisible" ],
+            "args": ["-d","-t","@adminLogin" ],
             "console":"integratedTerminal",
-            "runtimeVersion": "10.15.3",
+          //  "runtimeVersion": "10.15.3",
```
Comment (Parnswir) [inline .vscode/launch.json:14]: I'm not sure why the vscode project is checked in, but I guess you should revert your local changes before merging to keep it clean and not disturb other devs

Comment (CordlessWool) [reply .vscode/launch.json:14]: please also add it to .gitignore

Comment (d3n153) [reply .vscode/launch.json]: thanks for the finding, was rather unwanted, I reverted my changes

File: page-objects/adminLogin.js [hunk 2]
```diff
@@ -1,5 +1,6 @@
 'use strict';
 
+const { expect, assert } = require('chai');
```
[the remark this answers is not in the document]
Comment (d3n153) [reply page-objects/adminLogin.js]: fixed

**Pull request merged**